1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md

```diff
@@ -215,7 +215,7 @@ To demonstrate the vhost feature:
 
 1.  Find your [public agent IP](/docs/1.9/administration/locate-public-agent/). 
 
-1.  Modify the external nginx app (`nginx-external.json`) to point to your public agent IP. You can modify your app by using the DC/OS CLI or GUI.
+1.  Modify the external nginx app (`nginx-external.json`) to point to your public DNS name. You can modify your app by using the DC/OS CLI or GUI.
```

Comment:
Is it possible that this is the "fully qualified domain name (FQDN)"? I believe this is a more standard term.